ush/parsing_model_configure_FV3.sh

```diff
@@ -32,7 +32,7 @@ fhrot:                   ${IAU_FHROT:-0}
 
 dt_atmos:                ${DELTIM}
 calendar:                ${calendar:-'julian'}
-restart_interval:        ${restart_interval}
+restart_interval:        ${restart_interval} -1
```

Comment:
So `model_configure` still takes the explicit list, even though we can't use it for coupled? Why have one way of doing things when you can have three, I guess.

Reply:
what good does having restarts at a different time for FV3 than other components do?

Reply:
That's a good question. I think it is just not a high priority for UFS to align the two.